cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -60,8 +61,11 @@ const (
 	envHTTPProxy  = "HTTP_PROXY"
 	envNoProxy    = "NO_PROXY"
 
-	containerWorkingDir = "/tmp/kib"
-	windows             = "windows"
+	containerWorkingDir   = "/tmp/kib"
+	windows               = "windows"
+	envContainerEngine    = "KIB_CONTAINER_ENGINE"
```

Comment:
I think we should use the same approach here and in Konvoy.
Maybe a more generic env like `DKP_CONTAINER_ENGINE` or don't even expose this if its not being exposed in Konvoy, wdyt?

Reply:
Konvoy will not be using anything like this. It will default to docker if both are present.

Reply:
But why not the same behaviour in kib then @faiq? I agree this shouldn't be supported if we don't support it in konvoy for consistency.

Reply:
Please clarify the behavior that you're expecting here.

Reply:
if the host have both `podman` and `docker` installed then KIB will default to `docker`.  User wont be able to force KIB to use podman explicitly.  KIND uses optional variable [KIND_EXPERIMENTAL_PROVIDER](https://github.com/kubernetes-sigs/kind/blob/main/pkg/internal/runtime/runtime.go#L12) to provide similar option. 

if we do not want to set `DKP_CONTAINER_ENGINE` variable then we will need to document or assume that user will use `alias docker=podman` when both of them are installed on the host.

Reply:
I don't think we should document or "support" users using kind'ss env even in Konvoy since that may be removed/changed at any point.
So then to be consistent here it should also not have any env exposed and just default to `docker`.

Reply:
sounds good.

Reply:
removed using `KIB_CONTAINER_ENGINE` env variable. https://github.com/mesosphere/konvoy-image-builder/pull/789/commits/f82f0c51f5946d7b814506a8926c060b4c23a1e6

Reply:
KIND_EXPERIMENTAL_PROVIDER does not do anything when both podman and docker are present